spec/volcanic/authenticator/v1/identity_spec.rb

```diff
@@ -205,5 +205,15 @@
       it { expect(identities[1].id).to eq 4 }
       it { expect(identities[2].id).to eq 3 }
     end
+
+    context 'when find with name' do
+      subject(:identities) { identity.find(name: 'volcanic') }
+      it { expect(identities.first.name).to eq 'volcanic' }
+    end
+
+    context 'when find with name that has special character' do
+      subject(:identities) { identity.find(name: 'volcanic_ý') }
+      it { expect(identities.first.name).to eq 'volcanic_ý' }
+    end
```

Comment:
another test please to prove what happens when sending with a '+' character in the name, e.g. `volcanic+1234`

Reply:
@danleyden  done